osu.Desktop/osu.Desktop.csproj

```diff
@@ -25,6 +25,7 @@
   </ItemGroup>
   <ItemGroup Label="Package References">
     <PackageReference Include="Microsoft.NETCore.Targets" Version="5.0.0" />
+    <PackageReference Include="Mono.Posix.NETStandard" Version="5.20.1-preview" />
```

Comment:
Is there a reason for using the preview package? If not, a non-prerelease would be preferred.